src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/resthandler/RestRetryFailedManagedIndexAction.kt

```diff
@@ -95,55 +114,64 @@ class RestRetryFailedManagedIndexAction(
                                     builder.field(UPDATED_INDICES, listOfIndexMetaData.size)
                                 } else {
                                     failedIndices.addAll(listOfIndexMetaData.map {
-                                        FailedIndex(it.first.name, it.first.uuid, "failed to update IndexMetaData")
```

Comment:
np: can you check how long your max line length is? I think it looks ok for this to be a one liner and doesn't need to be broken down into multiple lines.

Reply:
My mistake, I didn't notice we changed the detekt configuration to a maximum of 150 lines. However, I think we should stick with 120 max lines. In my opinion, anything more than 120 affects readability and if we're going off of Kotlin's style conventions we shouldn't really be going over that number.

I went ahead and reverted my format changes to ```FailedIndex``` calls to use 150 lines as the max for now. I'm open to discussing this further if there are strong preferences on either side.